src/containers/AdminBulkScriptEditor.jsx

```diff
@@ -194,8 +194,9 @@ class AdminBulkScriptEditor extends Component {
             />
           </FormControl>
           <p style={{ fontStyle: "italic" }}>
-            Note: the text must be an exact match. For example, be careful of single quotes vs. double quotes: <span style={styles.code}>'</span> vs{" "}
-            <span style={styles.code}>’</span> )
+            Note: the text must be an exact match. For example, be careful of
+            single quotes vs. double quotes: <span style={styles.code}>'</span>{" "}
+            vs <span style={styles.code}>’</span>
```

Comment:
I removed the `)` when I adjusted some merge conflicts from 12.1 because it seemed out of place. Will note that if this seems to cause problems we should add it back